src/ui/widgets/Label/label.tsx

```diff
@@ -41,61 +42,57 @@ export const LabelComponent = (
   props: InferWidgetProps<typeof LabelProps>
 ): JSX.Element => {
   // Default labels to transparent.
-  const editedProps = {
-    ...props,
-    transparent: props.transparent ?? true
-  };
-  const style: CSSProperties = commonCss(editedProps);
   const {
+    transparent = true,
+    foregroundColor = diamondTheme.palette.primary.contrastText,
     textAlign = "left",
     textAlignV = "top",
     text = "",
-    rotationAngle,
+    rotationStep = 0,
     wrapWords
```

Comment:
This is an extra point, but `wrapWords` defaults to true in Phoebus. It would be good to add this in here